snapcraft/errors.py

```diff
@@ -98,7 +98,7 @@ def __init__(self, message: str, *, exit_code: int):
         self.exit_code = exit_code
         super().__init__(
             message,
-            resolution="Make sure the issues are addressed or ignore the isses in snapcraft.yaml.",
+            resolution="Make sure the issues are addressed or ignore the issues in snapcraft.yaml.",
```

Comment:
It's interesting that only now codespell started to complain about these misspelled words!